### PR TITLE
Settings action button added

### PR DESCRIPTION
**Pull request by rezaiyan**

### CODE REVIEW

File: quranHome/src/commonMain/kotlin/com/islamversity/quran_home/feature/home/QuranHomeProcessor.kt [hunk 1]
```diff
@@ -28,4 +28,13 @@ class QuranHomeProcessor(
             }
             .navigateTo(navigator)
     }
+
+    private val settingsClicked: FlowBlock<QuranHomeIntent, QuranHomeResult> = {
+        ofType<QuranHomeIntent.SettingsClicked>()
+            .map {
+                //TODO ADD TRANSITION
```
Comment (helloseyedjafari) [inline quranHome/src/commonMain/kotlin/com/islamversity/quran_home/feature/home/QuranHomeProcessor.kt]: what is a transaction?
do we really need it?

File: quranHome/src/main/res/layout/quran_home_view.xml [hunk 2]
```diff
@@ -10,11 +10,13 @@
     tools:showPaths="true">
 
     <ImageView
-        android:id="@+id/setting"
-        android:layout_width="40dp"
-        android:layout_height="40dp"
-        android:layout_margin="24dp"
-        android:padding="8dp"
+        android:id="@+id/settings"
+        android:layout_width="32dp"
```
Comment (helloseyedjafari) [inline quranHome/src/main/res/layout/quran_home_view.xml]: please use `dimens`

**Review by helloseyedjafari — COMMENTED** [review]
Well done

File: quranHome/src/commonMain/kotlin/com/islamversity/quran_home/feature/home/QuranHomeProcessor.kt (hunk 1 above)
Comment (rezaiyan) [reply quranHome/src/commonMain/kotlin/com/islamversity/quran_home/feature/home/QuranHomeProcessor.kt]: No

File: quranHome/src/main/res/layout/quran_home_view.xml (hunk 2 above)
Comment (rezaiyan) [reply quranHome/src/main/res/layout/quran_home_view.xml]: Sure

File: quranHome/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -6,4 +6,16 @@
         <item name="android:fontFamily">@font/vazir</item>
     </style>
 
+    <style name="TabLayout">
```
Comment (helloseyedjafari) [inline quranHome/src/main/res/values/styles.xml:9]: Cool

**Review by helloseyedjafari — APPROVED** [review]
Thanks

**Pull request merged**